pkg/config/datafileprojectconfig/config.go

```diff
@@ -82,6 +82,36 @@ func (c DatafileProjectConfig) GetAttributeID(key string) string {
 	return c.attributeKeyToIDMap[key]
 }
 
+// GetAttributes returns attributes slice
+func (c DatafileProjectConfig) GetAttributes() []entities.Attribute {
+	attributes := make([]entities.Attribute, 0, len(c.attributeMap))
+	for _, attr := range c.attributeMap {
+		attributes = append(attributes, attr)
+	}
+	return attributes
+}
+
+// GetEvents returns events slice
+func (c DatafileProjectConfig) GetEvents() []entities.Event {
+	events := make([]entities.Event, 0, len(c.eventMap))
+	for _, event := range c.eventMap {
+		events = append(events, event)
+	}
+	return events
+}
+
+// GetOptimizelyAudiences returns audiences slice
+func (c DatafileProjectConfig) GetOptimizelyAudiences() []entities.OptimizelyAudience {
+	optimizelyAudiences := make([]entities.OptimizelyAudience, 0, len(c.audienceMap))
+	for _, audience := range c.audienceMap {
```

Comment:
Can you confirm if audienceMap is made giving `typedAudiences` priority over `audiences`.